kasa/iot/iotstrip.py

```diff
@@ -246,11 +246,13 @@ async def _async_get_emeter_sum(self, func: str, kwargs: dict[str, Any]) -> dict
             ]
         )
 
-    async def erase_stats(self):
+    async def erase_stats(self) -> dict:
         """Erase energy meter statistics for all plugs."""
         for plug in self._device.children:
             await plug.modules[Module.Energy].erase_stats()
 
+        return {}
```

Comment:
As we are modifying the state of multiple child devices, I'm not sure what to return here? A combination of erase stats values or just an empty dict?